G1ANT.Addon.UI/G1ANT.Addon.UI/Addon.md

```diff
@@ -5,13 +5,13 @@
 
 | Name | Description |
 | ---- | ----------- |
-| [ui.click](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UIClickCommand.md) | Click control of the desktop application described by WPathStructure |
-| [ui.getrectangle](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UIGetRectangleCommand.md) | Get bounding box of the UI control described by WPathStructure |
-| [ui.gettext](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UIGetTextCommand.md) | Get text of the control of desktop application described by WPathStructure |
-| [ui.setfocus](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UISetFocusCommand.md) | Set focus on the control of desktop application described by WPathStructure |
-| [ui.setroot](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UISetRootCommand.md) | This command allows to attach and activate UI commands to desktop application |
-| [ui.settext](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UISetTextCommand.md) | Set text of the control of desktop application described by WPathStructure |
-| [waitfor.ui](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/WaitForUICommand.md) | Wait for the control of desktop application described by WPathStructure |
+| [ui.click](https://github.com/G1ANT-Robot/G1ANT.Addon/blob/master/G1ANT.Addon.UI/Commands/UIClickCommand.md) | This command clicks a UI element of a desktop application specified by WPath structure |
```

Comment:
I changed it in the description to:
This command clicks a desktop application UI element specified by WPath structure.

Reply:
But you changed it after I did this sync so you should approve.

Reply:
@Vegebutcher I merged develop to this branch and synced again so please approve :)